src/Entity/GroupManager.php

```diff
@@ -18,6 +18,10 @@
 use Sonata\DatagridBundle\Pager\PagerInterface;
 use Sonata\UserBundle\Model\GroupManagerInterface;
 
+if (!interface_exists(\Doctrine\Common\Persistence\ObjectManager::class)) {
```

Comment:
Can you add a comment to explain why we add this and when we can remove it ?

If one day we drop FOSUser, it will help.

Reply:
Done.